packages/big-design/src/components/Badge/Badge.tsx

```diff
@@ -5,9 +5,12 @@ import { MarginProps } from '../../mixins';
 import { StyledBadge } from './styled';
 
 export interface BadgeProps extends React.HTMLAttributes<HTMLSpanElement>, MarginProps {
+  label: string;
   variant?: 'danger' | 'secondary' | 'success' | 'warning';
 }
 
-export const Badge: React.FC<BadgeProps> = memo(({ className, style, ...props }) => <StyledBadge {...props} />);
+export const Badge: React.FC<BadgeProps> = memo(({ className, style, label, ...props }) => (
+  <StyledBadge {...props}>{label}</StyledBadge>
```

Comment:
What do you think about doing a type check for the label? 🤔

Reply:
I think that's wise.